src/scheduler/dfs.rs

```diff
@@ -21,18 +20,15 @@ pub struct DfsScheduler {
 }
 
 impl DfsScheduler {
-    /// Construct a new DFSScheduler with an optional bound on how many iterations to run
-    /// and an optional bound on the maximum depth to explore. A DFSScheduler can optionally
-    /// allow random data to be generated by the test (using `shuttle::rand`). Enabling random
-    /// data makes the DFS search incomplete, as it will not explore all possible values for
-    /// the random choices. To ensure determinism, each execution will use the same sequence of
-    /// random choices.
-    pub fn new(max_iterations: Option<usize>, max_depth: Option<usize>, allow_random_data: bool) -> Self {
+    /// Construct a new DFSScheduler. A DFSScheduler can optionally allow random data to be
```

Comment:
Could you add back the comment about the `max_iterations` argument?

Reply:
Oops, done